hazelcast-spring/src/main/resources/hazelcast-spring-4.0.xsd

```diff
@@ -3092,9 +3092,11 @@
                 <xs:restriction base="non-space-string">
                     <xs:enumeration value="random"/>
                     <xs:enumeration value="round-robin"/>
+                    <xs:enumeration value="custom"/>
```

Comment:
I am not sure about that but are we allowed to edit `.xsd` files of already released versions?

Reply:
@the-thing We should only change 4.1 xsd's.

Reply:
Rolled back 4.0 changes as suggested.